swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    """A child class of Item, representing aspects of an Item specific to Clothing"""
+    def __init__(self, category="Clothing", condition=0.0, age=None):
+        self.category = category
+        self.condition = condition
+        self.age = age
```

Comment:
`category` probably shouldn't be a parameter here since we don't want it to differ for different instances:

```suggestion
    """A child class of Item, representing aspects of an Item specific to Clothing"""
    def __init__(self, condition=0.0, age=None):
        self.category= "Clothing"
        self.condition = condition
        self.age = age
```